Week-1/Exercise1.2/src/index.css

```diff
@@ -0,0 +1,149 @@
+body {
+    font-family: Lato, sans-serif;
+}
+
+.container {
+    display: flex;
+    height: 100vh;
+    width: 100vw;
+    background-color: #f5f5f5;
+}
+
+.main {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    width: 100%;
+    height: 100%;
+}
+
+.song-list{
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    width: 90vw;
+    height: 100%;
+    overflow: auto;
+    margin: 10px;
+}
+table {
+  font-family: arial, sans-serif;
```

Comment:
Prefer using classes or ids to apply styling. Don't apply styles directly to HTML tags